README.md

```diff
@@ -15,6 +15,9 @@ https://annu12340.github.io/Javascript-Notes/
 - Passing by Value vs. Reference
 - Timer function
 - Truthy and Falsy values
+- concept of inheritence
+- inner classes
+- java IO stream
```

Comment:
Hey @arun2145 thanks for creating this pr 😃 
But I would like to keep the contents as short and simple as possible :)
So I don't think these topics are required
Also **java IO stream**, is not necessary since it is a js repo